hostdb/hostdb.go

```diff
@@ -2,15 +2,19 @@ package hostdb
 
 import (
 	"encoding/json"
+	"net"
 	"time"
 
 	"gitlab.com/NebulousLabs/encoding"
 	"go.sia.tech/renterd/internal/consensus"
+	rhpv2 "go.sia.tech/renterd/rhp/v2"
 	"go.sia.tech/siad/crypto"
 	"go.sia.tech/siad/modules"
 	"go.sia.tech/siad/types"
 )
 
+const InteractionTypeScan = "scan"
```

Comment:
I don't want to reify particular interaction types within the `hostdb` package. We can move this code (`IsScan`, `IsHost`, `IsOnline`, `LastKnownSettings`, `LatestHostScans`) into `bus`.